ui-tests/cypress/integration/administration/labels.spec.js

```diff
@@ -210,6 +210,20 @@ describe('Labels', () => {
             currentPage.getDialog().getBody().getStateInfo().should('exist').and('contain', testLabel.code);
         });
 
+        //SANITY TEST
+        //ENG-3238
+        it('When deletion is confirmed, modal should close and list should be updated', () => {
+            cy.labelsController().then(controller => controller.addLabel(testLabel.code, testLabel.name));
+            cy.wrap(testLabel).as('labelToDelete');
+            currentPage = openLabelsPage();
+            currentPage.getContent().getKebabMenu(testLabel.code).open().clickDelete();
+            currentPage.getDialog().confirm();
+            cy.wrap(null).as('labelToDelete');
+            cy.wait(1000);
```

Comment:
Please, add a comment when you use the _wait_; we should be able to remove them eventually and it's easier if we track why we've used it

Reply:
I removed the waits, since, upon further tries, they don't seem to be needed